### PR TITLE
prevent search indexing on readthedocs

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ build:
   commands:
     - python -m pip install --upgrade --no-cache-dir pip setuptools
     - python -m pip install --upgrade -r ./requirements.in -c ./requirements.txt
+    - sed -i '12 s/<!--\(.*\)-->/\1/' ./themes/ansible-community/templates/base.tmpl
```
Comment (GregSutcliffe) [inline .readthedocs.yaml:15]: Why enable/disable as a sed? Can we not just commit the `meta` without the comment block?

Comment (oraNod) [reply .readthedocs.yaml]: We don't want the `meta` in the final live version do we? I thought this way we can control when that gets inserted.

Comment (GregSutcliffe) [reply .readthedocs.yaml]: Well sure, but we could just open an issue to remove it when the site finally goes live, no need to overcomplicate it ;)

Comment (oraNod) [reply .readthedocs.yaml]: But we'd still want the preview site on RTD wouldn't we?

Comment (GregSutcliffe) [reply .readthedocs.yaml]: clarifying the "possibly" - once the site is live is there any value in the preview? I see value in the PR previews, but the main branch should be on the live site anyway, right?

Comment (oraNod) [reply .readthedocs.yaml]: Good point about forks and other deployments. It's probably also more reliable to remove the line as part of the "prod" build/deploy. If someone were to modify `base.tmpl` and the `meta` tag didn't stay on line 12 then the sed thing breaks.

As for the value of the readthedocs project after the final site is launched, I think we should keep it around for dev purposes. We might want to add a new band to the homepage or ditch nikola for pelican or whatever else. Having a test site makes a lot of sense.

**Comment by oraNod** [comment]
Hmm.. I might have been seriously overcomplicating things and should have let nikola do the work. I made some changes in the conf.py and we can likely drop the `meta` stuff from the template.

File: conf.py [hunk 2]
```diff
@@ -1360,7 +1360,7 @@
 USE_BUNDLES = False
 
 # Plugins you don't want to use. Be careful :-)
-# DISABLED_PLUGINS = ["render_galleries"]
+DISABLED_PLUGINS = ["robots"]
```
Comment (GregSutcliffe) [inline conf.py:1363]: Wait, you want to disable Rbots? That sounds like the opposite of what we want, have I misunderstood something here?

Comment (oraNod) [reply conf.py]: Urgh, I think it was my misunderstanding. This isn't particularly well documented but I was getting a warning that seemed like it was related to the exclusions. Disabling robots seemed to fix that while still generating `output/robots.txt` and the sitemap as expected.

On second look that warning seems to have gone away and was in fact related to the value of `SITE_URL`. Not sure if the fact nikola was still generating robots.txt with this is a bug or not.

**Review by GregSutcliffe — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks for the review @GregSutcliffe

**Pull request merged**